chessington/engine/board.py

```diff
@@ -77,3 +78,11 @@ def move_piece(self, from_square, to_square):
             self.set_piece(to_square, moving_piece)
             self.set_piece(from_square, None)
             self.current_player = self.current_player.opponent()
+
+    def is_inside_bounds(self, square):
+        """
+        Checks if the given square is inside the bounds of the board
+        """
+        if 0 <= square.row <= 7 and 0 <= square.col <= 7:
+            return True
+        return False
```

Comment:
1. Define constants instead of hardcoded values
2. You can return the statement instead of using an` if`.